tests/cpp/test_translate_mma.cpp

```diff
@@ -192,6 +209,7 @@ TEST_F(CombineMulSumAsMmaTest, AmpereMulSumToMatmul_MultipleBroadcasts) {
 // pair with a mma op, we are able to schedule it as we did with
 // a fusion that had a mma op to begin with.
 TEST_P(CombineMulSumAsMmaTestWithLayout, AmpereMulSumToMatmul_Schedule) {
+  NVFUSER_TEST_CUDA_ARCH_RANGE_GUARD(7, 5, 9, 0);
```

Comment:
These tests will be unguarded once #3579 is merged and we can automatically accept and schedule these fusions.